src/components/inventoryCard/__tests__/__snapshots__/inventoryCardContext.test.js.snap

```diff
@@ -3,12 +3,15 @@
 exports[`InventoryCardContext should expect specific sort properties: sort properties 1`] = `
 {
   "BILLING_PROVIDER": "billing_provider",
-  "CORES": "Cores",
+  "CORES": "cores",
+  "CORE_HOURS": "core_hours",
   "CORE_SECONDS": "Core-seconds",
-  "INSTANCE_HOURS": "Instance-hours",
+  "HARDWARE": "hardware_type",
+  "INSTANCE_HOURS": "instance_hours",
   "LAST_SEEN": "last_seen",
+  "MEASUREMENT": "measurement_type",
   "NAME": "display_name",
-  "SOCKETS": "Sockets",
+  "SOCKETS": "sockets",
```

Comment:
required:
confirm this isn't going to be an issue with the sort-types check

Reply:
confirmed, it does appear this causes issues with `Cores` vs `cores` and `Sockets` vs `sockets`.